build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     ext.wordPressUtilsVersion = 'develop-f98c841225fbc61031f32426aaef4e347b34f717'
     ext.wordPressLoginVersion = '0.0.3'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.59.1'
+    ext.gutenbergMobileVersion = '3844-8992fef55c6991b7f24651f3ac65ac11c38550f2'
```

Comment:
Before merging, we'll need to update the `gutenbergMobileVersion` ref to the release tag once it is created.